src/components/Modal/Translate.vue

```diff
@@ -119,6 +125,15 @@ export default {
 		}
 	},
 	computed: {
+		inputValue: {
+			get() {
+				return this.input
+			},
+			set(event) {
+				this.input = event.target.value
+				this.autosize()
+			},
+		},
```

Comment:
If we want to resize on input, we can watch the `input` property, so it won't be important how the value has changed for the resize.